lib/easy_listview.dart

```diff
@@ -155,7 +155,7 @@ class EasyListViewState extends State<EasyListView> {
   int _footerCount() => (_hasFooter() || widget.loadMore) ? 1 : 0;
 
   int _dataItemCount() =>
-      _hasDivider() ? widget.itemCount * 2 - 1 : widget.itemCount;
+      _hasDivider() && widget.itemCount > 0 ? widget.itemCount * 2 - 1 : widget.itemCount;
```

Comment:
The fix should be:
_hasDivider() ? widget.itemCount * 2  : widget.itemCount;
Otherwise the '-1' is causing other issues, specifically the last data item is never shown when the headerSliverBuilder is non-null.